### PR TITLE
fix: restructuring of guides and tutorials under misc resources section

### LINKED ISSUE #2866: "Managing Jenkins X" not accessible via the navigation
Screenshot 2020-06-11 at 13 25 46

### PR DESCRIPTION
**Pull request by hferentschik**
- make managing JEnkins X visible in navigation again
- split guides&tutorials into two seperate section
- changes size of iframe for gitpitch presentation

fixes #2866

### CODE REVIEW

File: content/en/docs/resources/_index.md [hunk 1]
```diff
@@ -9,21 +9,18 @@ aliases:
...
 ---
 
-# Presentations
-We have developed a presentation to help you introduce Jenkins X to your team, we will be adding more soon! Right from this site, you can present.
+## Presentations
 
-<iframe src="https://gitpitch.com/sharepointoscar/presentations/master?p=talks/intro-to-jenkins-x#/" frameborder="0" width="1280" height="749"  webkitallowfullscreen="true"></iframe>
+We have developed a presentation to help you introduce Jenkins X to your team, we will be adding more soon! Right from this site, you can present.
 
+<iframe src="https://gitpitch.com/sharepointoscar/presentations/master?p=talks/intro-to-jenkins-x#/" frameborder="0" scrolling="no" allowtransparency="true" allowfullscreen="true" id="emb_5jtc6n" height="401" width="600" style="border: none; position: relative; visibility: visible; display: block; margin: 0px; padding: 0px; max-width: 100%; min-width: 200px;"></iframe>
```
Comment (ankitm123) [inline content/en/docs/resources/_index.md:16]: Not related to this PR, but the gitpitch slides (page 11) say `jx` runs on aks, eks and gke, may be remove aks?

Comment (hferentschik) [reply content/en/docs/resources/_index.md:16]: that would be a something for @sharepointoscar since this is his presentation

Comment (sharepointoscar) [reply content/en/docs/resources/_index.md:16]: @ankitm123 I updated the deck, removed Azure since it is WIP.

**Comment by sharepointoscar** [comment]
> /lgtm

The formatting looks bad, has anyone looked at this?
image

@MarckK @hferentschik

**Pull request merged**